l10n_co_geo/data/res.country.csv

```diff
@@ -0,0 +1,254 @@
+"id","code_dian","name"
```

Comment:
It's because of the `code_dian` column, which is neede. Is the way we do it valid?